lib/cisco_node_utils/bgp.rb

```diff
@@ -152,6 +158,17 @@ def bestpath_med_non_deterministic
       config_get('bgp', 'bestpath_med_non_deterministic', @get_args)
     end
 
+    # nsr Setter
+    def nsr=(enable)
+      if platform == :nexus || @vrf != 'default'
+        fail Cisco::UnsupportedError.new('bgp', 'nsr', 'set', 'nsr is not ' \
```

Comment:
To do this (which I'm in favor of, btw) you'll have to remove the nexus check (which is fine, as you've excluded Nexus in the YAML so the config_set() below will automatically raise an UnsupportedError.
```
if platform == :ios_xr && @vrf != 'default'
  fail Cisco::UnsupportedError.new(...
```